Quantum-Control-Applications/Superconducting/Multi Flux Tunable transmon/Use Case 1 - Two qubit gate optimization with cryoscope/readme.md

```diff
@@ -0,0 +1,192 @@
+# Two-qubit SWAP spectroscopy improved with pre-distortion digital filters
+
+_Authors: Kevin Villegas Rosales & Theo Laudat_
+
+*Important note: the code in this folder is the exact code that was used for running this measurement*
+*and it is tailored for a very specific setup and software environment. Thus, the code is here only for insipiration.*
+
+The goal of this use-case is threefold:
+1. Implement Cryoscope [[1](#1), [2](#2)] to measure the step response of the flux line
+2. Design suitable IIR and FIR filters to correct for distortion occuring along the flux line
+3. Perform two qubit SWAP spectroscopy without and with predistortion filters to characterize the corrections.
+
+
+
+## 1. Experimental set-up and required calibrations
+
+<img align="right" src="setup.PNG" alt="drawing" width="400"/>
+
+### 1.1 Experimental set-up
+The chip contains 5 2D-transmons, where some of them have nearest-neighboor connectivity, 
+with flux tunable lines.
+
+The experimental setup specific to this experiment consists of two flux tunable transmons (with a SQUID loop as nonlinear inductor and a flux line next to it) coupled to readout resonators located on a common transmission line.
+The general scheme is represented on the figure on the right.
+
+The qubits are controlled with IQ signals generated by the OPX and upconverted with an external local oscillator using calibrated IQ mixers.
+
+The readout resonators are pumped with an intermediate frequency generated by the OPX and upconverted with another external local oscillator using an IQ mixer.
+The transmitted signal is measured by the OPX after downconversion with an IQ mixer and the local oscillator signal.
+
+The flux lines are controlled by two additional analog outputs of the OPX.
+
+In principle, the program detailed below can be easily modified if a different set-up is to be used.
+
+### 1.2 Calibration steps prior to Cryoscope and two-qubit SWAP spectroscopy
+
+Before running Cryoscope, several calibration steps are required.
+
+* The first calibration to run is the 2D readout resonator spectroscopy (flux amplitude and readout frequency). This spectroscopy will show the resonator frequency as a function of flux biasing, the qubit flux insensitive point (qubit frequency independent of flux bias), the qubit zero-frequency point, and the avoided crossing when the qubit and resonator have the same frequency. For Cryoscope, we want to set the flux line to the qubit flux insensitive point and bias it around $\phi_0/4$ during the sequence.   

```

Comment:
Do we have a nice figure showing this 2D scan? It would be nice, with an indicaiation of the features

Reply:
I don't, maybe @KevinAVR has one ?

Reply:
I don't have either and the system is not in our hand anymore -- next time